src/components/Selector/Selector.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme }) => css`
+  label: selector;
+  cursor: pointer;
+  padding: ${theme.spacings.mega};
+  border-radius: ${theme.borderRadius.mega};
+  border: 1px solid ${theme.colors.n300};
+  background-color: 1px solid ${theme.colors.n100};
+  margin-bottom: ${theme.spacings.mega};
```

Comment:
I see that having a bottom margin makes sense on mobile. How does this work on larger screens? Don't selectors need a `margin-right` then?